etna/articles/blocks.py

```diff
@@ -216,7 +216,6 @@ class QuoteBlock(SectionDepthAwareStructBlock):
     the content depth.
     """
 
-    heading = blocks.CharBlock(required=False, max_length=100)
```

Comment:
Task: Update the doc-string above, if it no longer has heading it does not make sense (Double check the FE templating too).